src/network/postgres_protocol_handler.cpp

```diff
@@ -373,15 +374,16 @@ ProcessResult PostgresProtocolHandler::ExecQueryMessage(InputPacket *pkt, const
         }
         // ExecuteStatment
         std::vector<type::Value> param_values;
-        param_values_ = param_values;
+        traffic_cop_->param_values_ = param_values;
         bool unnamed = false;
-        std::vector<int> result_format(statement_->GetTupleDescriptor().size(), 0);
+        std::vector<int> result_format(traffic_cop_->statement_->GetTupleDescriptor().size(), 0);
         result_format_ = result_format;
         // should param_values and result_format be local variable?
         // should results_ be reset when PakcetManager.reset(), why results_ cannot be read?
         auto status =
-            traffic_cop_->ExecuteStatement(statement_, param_values_, unnamed, nullptr, result_format_,
-                                           results_, rows_affected_, error_message_, thread_id);
+            traffic_cop_->ExecuteStatement(traffic_cop_->statement_, traffic_cop_->param_values_, unnamed, nullptr,
```

Comment:
Same problem here. Use setter to set the member values first, and then call the function without passing as the argument but rather use that directly inside the function.